tests/TestCase/Cache/Engine/ApcEngineTest.php

```diff
@@ -286,4 +286,24 @@ public function testGroupClear()
         $this->assertTrue(Cache::clearGroup('group_b', 'apc_groups'));
         $this->assertFalse(Cache::read('test_groups', 'apc_groups'));
     }
+
+    /**
+     * Test that failed add write return false.
```

Comment:
I fixed the comments. As for consistency, I'm going to look at extracting the common interface tests into a trait. The duplication across the cache backend tests is pretty annoying.